src/ui/QuoteMessageInput/QuoteMessageThumbnail.tsx

```diff
@@ -24,12 +25,51 @@ export default function QuoteMessageThumbnail({ message }: Props): ReactElement
   if (!isFileMessage(message) && !isMultipleFilesMessage(message) || isVoiceMessage(message as FileMessage)) {
     return null;
   }
-  const thumbnailUrl: string = isFileMessage(message)
-    ? (getMessageFirstFileThumbnailUrl(message) || (isImageMessage(message as FileMessage) && getMessageFirstFileUrl(message)))
-    : (getMessageFirstFileThumbnailUrl(message)
-      || isImageFileInfo((message as MultipleFilesMessage).fileInfoList[0])
-      && getMessageFirstFileUrl(message));
-  if ((isThumbnailMessage(message) || isMultipleFilesMessage(message)) && thumbnailUrl) {
+  let thumbnailUrl = getMessageFirstFileThumbnailUrl(message);
+  if (!thumbnailUrl) {
+    if (
+      message.isFileMessage?.()
+      && (
+        isImageMessage(message as FileMessage)
+        || isVideoMessage(message as FileMessage)
+      )
+    ) {
+      thumbnailUrl = getMessageFirstFileUrl(message);
+    } else if (
+      message.isMultipleFilesMessage?.()
+      && (
+        message.fileInfoList.length > 0
+        && isImageFileInfo((message as MultipleFilesMessage).fileInfoList[0])
+      )
+    ) {
+      thumbnailUrl = message.fileInfoList[0].url;
+    }
+  }
+  if (thumbnailUrl) {
+    return (
+      <ImageRenderer
+        className={componentClassname}
+        url={thumbnailUrl}
+        alt={getMessageFirstFileType(message)}
+        width="44px"
+        height="44px"
+        fixedSize
+      />
+    );
+  }
+  if (isVideoMessage(message as FileMessage) && thumbnailUrl) {
```

Comment:
it seems not reachable if `thumbnailUrl` is available.

Reply:
Thanks for the catch. It is a duplicate